docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1729,6 +1729,16 @@ The limit protects the system’s stability from potential abuse or mistakes. To
 Invalid series are skipped during the ingestion, and valid series within the same request are ingested.
 {{< /admonition >}}
 
+### err-mimir-max-label-names-per-info-series
+
+This non-critical error occurs when Mimir receives a write request that contains an info series with a number of labels that exceed the configured limit.
+An info series is a series where the metric name ends in `_info`.
```

Comment:
Is `info series` an abbreviation for `information series` or is it really just called an `info series`? If it's an abbreviation, we should spell it out except when using a code-formatted label.

Reply:
not an abbreviation, these are called info series/metrics everywhere